### ISSUE TITLE
Excluding some packages on the fly

### ISSUE THREAD

**Issue by xfzv** [fosskers/aura#880]
`paru` and `yay` allow to exclude packages on the fly during the system update:

- `paru`
![paru](https://github.com/user-attachments/assets/ff1fa18d-fa96-4293-a44b-2c98464f2109)

- `yay`
![yay](https://github.com/user-attachments/assets/3f55541e-a859-4be4-bc36-d8bc70cbb3d9)

This is convenient if the user wants to postpone some packages updates but still wants to update the others.
This is faster and more convenient than:

1. Cancelling the system update
2. Adding package(s) to `~/.config/aura/config.toml`
3. Running the system update again
4. Remove the excluded packages from `~/.config/aura/config.toml`

Considering partial upgrades aren't supported by Arch, I guess this option could be disabled by default.

**Comment by fosskers** [comment]
One thing to point out is that the `--ignore` flag is available. That at least would allow you to avoid the cycle of editing your config file.

**Comment by RubenKelevra** [comment]
@fosskers yeah, but this gets annoying pretty quickly. Say there's a Haskell update again, and I like to ignore like 50 packages at once.

**Comment by fosskers** [comment]
The secret to that, of course, is to avoid Haskell packages at all costs on Arch Linux :rofl:
